### PR TITLE
Feature/oe 164 add set sits throughout system

### PR DESCRIPTION
**Pull request by Ron-Junior**
## Goal

Add new input set sits on update numbers

## Changeset

- Add new input to insert set sits
- change overview cards: remove set closes card and add SG on sits and closes cards

## Tests

- [ ] Go to one energy project
- [ ] open number tracker page 
- [ ] see if sits and closes card are showed correctly
  obs: in cards sits and closes the labels means
     - Sg is the closes ou sits
     - set is the set closes or set sits

## Review

For the submitter, initial self-review:

- [ ] Reviewed the test cases added for completeness and possible points for discussion
- [ ] Check the scope of the changeset - is everything in the diff required for the pull request?

For the pull request reviewer(s), this changeset has been reviewed for:

- [ ] Consistency between the changeset and the goal stated above
- [ ] Usage friction - is the proposed change in usage cumbersome or complicated?
- [ ] Complexity - is the proposed change introducing any unjustified complexity? 
- [ ] Thoroughness of added tests and any missing edge cases

### CODE REVIEW

File: app/Http/Livewire/NumberTracker/DailyEntry.php [hunk 1]
```diff
@@ -57,6 +57,7 @@ public function getUsers($dateSelected)
                 'daily_numbers.doors',
```
Comment (iammayron) [inline app/Http/Livewire/NumberTracker/DailyEntry.php:57]: image

File: app/Http/Livewire/NumberTracker/NumberTrackerDetail.php [hunk 2]
```diff
@@ -92,7 +92,7 @@ public function getTrackerNumbers()
             })
             ->join('offices', 'users.office_id', '=', 'offices.id')
             ->select([DB::raw("users.first_name, users.last_name, daily_numbers.id, daily_numbers.user_id, SUM(doors) as doors,
-                    SUM(hours) as hours,  SUM(sets) as sets,  SUM(sits) as sits,  SUM(set_closes) as set_closes, SUM(closes) as closes")]);
+                    SUM(hours) as hours,  SUM(sets) as sets, SUM(set_sits) as set_sits,  SUM(sits) as sits,  SUM(set_closes) as set_closes, SUM(closes) as closes")]);
 
         $queryLast = clone $query;
```
Comment (iammayron) [inline app/Http/Livewire/NumberTracker/NumberTrackerDetail.php:97]: You don't need to use that "clone" word, you could simply use `$queryLast = $query;`

**Review by iammayron — CHANGES_REQUESTED** [review]

**Review by iammayron — APPROVED** [review]
